task16/index.js

```diff
@@ -261,18 +83,74 @@ const renderTabContainer = (value) => {
 const createArticleTitle = ({ article }) => {
   const articleTitleFragment = document.createDocumentFragment();
   for (let i = 0; i < article.length; i++) {
-    const articleTitleItem = createElementWithClassName("li", "article-title-item");
-    const articleWrapper = createElementWithClassName("article", "article-wrapper");
+    const articleTitleItem = createElementWithClassName(
+      "li",
+      "article-title-item"
+    );
+    const articleWrapper = createElementWithClassName(
+      "article",
+      "article-wrapper"
+    );
     const articleLink = createElementWithClassName("a", "article-link");
     const articleTitle = createElementWithClassName("h1", "article-title");
     articleTitle.textContent = article[i].title;
     articleLink.href = "#";
 
-    articleTitleFragment.appendChild(articleTitleItem).appendChild(articleWrapper).appendChild(articleLink).appendChild(articleTitle);
+    articleTitleFragment
+      .appendChild(articleTitleItem)
+      .appendChild(articleWrapper)
+      .appendChild(articleLink)
+      .appendChild(articleTitle);
+
+    renderNewArrivals(article[i], articleLink);
+    renderCommentCount(article[i], articleLink);
   }
   return articleTitleFragment;
 };
 
+const renderNewArrivals = (article, articleLink) => {
+  const articleDate = elapsedTime(article.date);
+  const newArrivalsLimiter = 3;
+  if (articleDate <= newArrivalsLimiter) {
+    articleLink.appendChild(createNewArrivals());
+  }
+};
+
+const renderCommentCount = (article, articleLink) => {
+  const commentLength = article.comment.length;
+  if (commentLength > 0) {
+    articleLink.appendChild(createCommentCount(article));
+  }
+};
+
+const elapsedTime = (articlePostedDate) => {
```

Comment:
IMO
IMO
Here we are comparing two dates and returning the number of days.

If so, I don't think the function name is appropriate.
As it stands, I feel that it would be assumed to return the time.

Reply:
Thank you for pointing this out. It was indeed odd. I'm glad I was able to fix it.
6c7fbc4

Reply:
IMO
I think what we got from the comparison is the difference.
How about `getDiffDays`?

Reply:
Sorry for not naming it well. Thanks for pointing that out. I will learn more about naming.
4ec978e